js/gate.js

```diff
@@ -2680,7 +2680,7 @@ module.exports = class gate extends Exchange {
         const gtFee = this.safeString (trade, 'gt_fee');
         const pointFee = this.safeString (trade, 'point_fee');
```

Comment:
@ttodua to avoid having to remember to check against every operation (costly since it's a string comparison), why not use `omitZero` upon parsing the value?

```Javascript
const pointFee = this.omitZero (this.safeString (trade, 'point_fee'));
````

Reply:
can you please re-review the code? i think you are commenting on old code (currently, we are removing that comparison at all).

Reply:
@ttodua You're right, sorry my bad